test/functional/feature_fee_estimation.py

```diff
@@ -38,9 +38,9 @@ def small_txpuzzle_randfee(
     # It's best to exponentially distribute our random fees
     # because the buckets are exponentially spaced.
     # Exponentially distributed from 1-128 * fee_increment
-    rand_fee = float(fee_increment) * (1.1892 ** random.randint(0, 28))
+    rand_fee = satoshi_round(fee_increment * Decimal(1.1892 ** random.randint(0, 28)), rounding=ROUND_DOWN)
```

Comment:
Because I cannot directly multiply a` Decimal object (fee_increment)` with a `float (the result of 1.1892 ** random.randint(0, 28))`

results in an aror
**TypeError: unsupported operand type(s) for *: 'decimal.Decimal' and 'float'**

Reply:
@naiyoma  thats because `1.1892` is in float, I think you should instead convert it to decimal and do the exponentiation in decimal?
```suggestion
    rand_fee = satoshi_round(fee_increment * (Decimal('1.1892') ** random.randint(0, 28)), rounding=ROUND_DOWN)

```

Reply:
thanks, makes sense to convert `(Decimal('1.1892')` first.